t/porting/bidi-control.t

```diff
@@ -42,11 +46,13 @@ unless (@files) {
 }
 
 FILE: for my $file (@files) {
-  open my $fh, '<', "$file"
+  next if $SKIP_FILE{$file};
```

Comment:
Ah, I see you found an example, and invented a mechanism. The `if (0)` block seems quite redundant now.